circle.yml

```diff
@@ -389,6 +389,9 @@ jobs:
       - run:
           name: Build package
           command: make apackage
+      - run:
+          name: Build release Test App
+          command: make android
```

Comment:
I'm wondering if this `make android` will recompile and obfuscate the aar? if so we need to make sure that this step doesn't execute when we are doing a release with `run-android-upload-archives`.

Reply:
It only assembles `MapboxGLAndroidSDKTestApp` module. SDK `aar`s are kept intact.

Reply:
I would have though that this would also result in compiling the [SDK module](https://github.com/mapbox/mapbox-gl-native/blob/master/platform/android/MapboxGLAndroidSDKTestApp/build.gradle#L55)? 
It seems you are certain about this so lets go for it!